packages/formstr-app/src/components/Header/configs.tsx

```diff
@@ -1,33 +1,57 @@
 import { Link } from "react-router-dom";
-import { Button } from "antd";
+import { Button, Dropdown, Menu } from "antd";
 import {
   SearchOutlined,
   PlusOutlined,
   InfoCircleOutlined,
+  DownOutlined,
+  FormOutlined,
+  BarChartOutlined
 } from "@ant-design/icons";
 import { ROUTES } from "../../constants/routes";
 
 export const HEADER_MENU_KEYS = {
   PUBLIC_FORMS: "PUBLIC_FORMS",
+  PUBLIC_POLLS: "PUBLIC_POLLS",
   USER: "USER",
   CREATE_FORMS: "CREATE_FORMS",
   HELP: "HELP",
+  GLOBAL: "GLOBAL",
 };
 
+// Define the global dropdown menu items
+const globalItems = [
+  {
```

Comment:
Instead of in the header button, let's have this drop down in the global feed itself